File: TruffleCommon/src/main/java/io/synople/truffle/common/model/Ticket.java
package io.synople.truffle.common.model;


import java.util.List;

public class Ticket {

    private String id;
    private List<String> items;
    private String customerId;
    private String vendorId;
    private String time;
    private double amount;

    public Ticket(String id, List<String> items, String customerId, String vendorId, String time, double amount) {
        this.id = id;
        this.items = items;
        this.customerId = customerId;
        this.vendorId = vendorId;
        this.time = time;
        this.amount = amount;
    }
}

File: TruffleCommon/src/main/java/io/synople/truffle/common/model/Ticket.kt
package io.synople.truffle.common.model


import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import java.util.ArrayList
import java.util.UUID

@Parcelize
data class Ticket(
    var id: String = "",
    var items: MutableList<Item> = mutableListOf(),
    var customerId: String = "",
    var vendorId: String = "",
    var time: String = "",
    var amount: Double = 0.0
) : Parcelable
